pkg/storage/trb_request_form.go

```diff
@@ -15,7 +15,6 @@ import (
 
 // UpdateTRBRequestForm updates a TRB request form record in the database
 func (s *Store) UpdateTRBRequestForm(ctx context.Context, form *models.TRBRequestForm) (*models.TRBRequestForm, error) {
```

Comment:
@DylanSpOddball I discussed this with @ClayBenson94 when doing the TRB attendees PR, we arrived at keeping the SQL inline - there's a case to be made for inline SQL over separate files, happy to discuss further if you want. The one place I did add a separate file here was for the creation, because the TRB request SQL used it, and it seemed reasonable to at least follow that pattern within the same function that was already using it.